dbt/adapters/databricks/python_models/python_submissions.py

```diff
@@ -142,12 +142,9 @@ def submit(self, compiled_code: str) -> None:
                 config["existing_cluster_id"] = self.cluster_id
             self._submit_through_notebook(compiled_code, self._update_with_acls(config))
         else:
-            context_id = self.api_client.command_contexts.create(self.cluster_id)
```

Comment:
OK so the context is created for every submission of a compiled code so probably fine to just move to commands execution?